### ISSUE TITLE
Failed to read static data file when run with docker-compose

### ISSUE THREAD

**Issue by AbyssLink** [heartexlabs/label-studio#669]
**Describe the bug**
Can't get static file from Nginx when run with docker-compose.

**To Reproduce**
1. Run with `docker-compose up -d`.
2. Sign in to admin-page, create a "Sound Event Detection" project, then click import data.
3. Open project view.
4. See error: "Wow! Error while processing audio. Check media format and availability."

**Expected behavior**
Static file load successfully.

**Screenshots**
![Xnip2021-03-18_13-23-53](https://user-images.githubusercontent.com/26038696/111577526-68ca0100-87ed-11eb-8b97-1902c5c722fe.jpg)

**Environment (please complete the following information):**
 - OS: macOS 11.2.3
 - Docker Version: 20.10.5
 - Label Studio Version 1.0.0

**Comment by AbyssLink** [comment]
This may be an error caused by the wrong path.
I temporarily modified the following content to make it work.

```shell
diff --git a/docker-compose.yml b/docker-compose.yml
@@ -10,6 +10,7 @@ services:
       - app
     volumes:
       - static:/label-studio/label_studio:rw
+      - ./files:/label-studio/files:rw
       - ./nginx/:/etc/nginx/conf.d/:ro
     command: nginx -g "daemon off;"
```

```shell
diff --git a/nginx/ls.conf b/nginx/ls.conf
@@ -18,7 +18,7 @@ server {
     }

     location /data/upload {
-        alias /label-studio/label_studio/core/media/upload/;
+        alias /label-studio/files/media/upload;
     }
```

![Xnip2021-03-18_13-30-54](https://user-images.githubusercontent.com/26038696/111578513-3f11d980-87ef-11eb-8bff-0040161b55c9.jpg)

**Comment by niklub** [comment]
Great! That's an error in nginx config paths for sure. @antonht can you please make a fix as @AbyssLink suggested?

**Comment by mishra011** [comment]
I am having the same issue. 





GET /api/dm/views/1/tasks?page=1&page_size=30&project=1 HTTP/1.1" 200 2030 "http://100.2.1.49/projects/1/data?tab=1" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36" "-"
nginx_1  | 10.10.7.15 - - [31/Mar/2021:10:04:30 +0000] "GET /data/upload/valid__en-in__000001__7270__8k.wav HTTP/1.1" 404 555 "http://100.2.1.49/projects/1/data?tab=1" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36" "-"
nginx_1  | 2021/03/31 10:04:30 [error] 22#22: *33 open() "/label-studio/files/media/upload/valid__en-in__000001__7270__8k.wav" failed (2: No such file or directory), client: 10.10.7.15, server: _, request: "GET /data/upload/valid__en-in__000001__7270__8k.wav HTTP/1.1", host: "100.2.1.49", referrer: "http://100.2.1.49/projects/1/data?tab=1"
nginx_1  | 10.10.7.15 - - [31/Mar/2021:10:04:30 +0000] "GET /data/upload/valid__en-in__000007__4450__8k.wav HTTP/1.1" 404 555 "http://100.2.1.49/projects/1/data?tab=1" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/89.0.4389.90 Safari/537.36" "-"
nginx_1  | 2021/03/31 10:04:30 [error] 22#22: *33 open() "/label-studio/files/media/upload/valid__en-in__000007__4450__8k.wav" failed (2: No such file or directory), client: 10.10.7.15, server: _, request: "GET /data/upload/valid__en-in__000007__4450__8k.wav HTTP/1.1", host: "100.2.1.49", referrer: "http://100.2.1.49/projects/1/data?tab=1"

**Comment by makseq** [comment]
@mishra011 Could you check `pip install label-studio==1.0.1rc1`, I hope we've fixed it there.

**Comment by niklub** [comment]
Hey, this problem is supposed to be fixed following `label-studio>=1.0.1`. Please feel free to reopen the issue in case it happens again.
